### PR TITLE
fix for #2426

### PR DESCRIPTION
**Pull request by fviale**
- The install script asks various configuration questions to the user
- It copies the scheduling distribution to a specified folder
- It eventually creates the user account starting the scheduler
- It installs the service.

### CODE REVIEW

**Comment by lpellegr** [comment]
I have tried the script with success on Fedora 23. 

However, I noticed that _install.sh_ script is currently installing the distribution from the sources. It means that a user who  clones the scheduling repository and uses the install script will install a ProActive version that does not deploy Scheduler and RM web portals. Asking for the distribution to install (i.e. a path to the ZIP file to use or support for remote downloading of a distribution) would be more practical and help us to upgrade an existing installation.

File: tools/install.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    esac
+done
+
+chown -R $USER:$USER $PA_ROOT/$CURRENT_PA_DIR
+
+# installation of the service script
+
+cp $SCRIPT_DIR/proactive-scheduler /etc/init.d/
+
+read -e -p "Protocol used by the proactive server: " -i "http" PROTOCOL
```
Comment (lpellegr) [inline tools/install.sh]: You said that it makes sense to specify the options which are allowed between square brackets but you don't did it. Did you forget?

File: tools/install.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    case $yn in
+        Yes ) 
+		useradd -m $USER;
+		passwd $USER; 
+		break;;
+        No ) break;;
+    esac
+done
+
+chown -R $USER:$USER $PA_ROOT/$CURRENT_PA_DIR
```
Comment (lpellegr) [inline tools/install.sh]: Group to use is not asked by installation script as we discussed.

Comment (fviale) [inline tools/install.sh]: There is a version problem. This code didn't include my changes.

**Comment by lpellegr** [comment]
@fviale Available protocols are not displayed with `install.sh`. Otherwise it looks okay for me.

**Pull request merged**